<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gengxin</class>
 <widget class="QDialog" name="Gengxin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labUp">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labMv">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBtnTerm">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>141</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>强制终止更新</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progBar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>161</width>
     <height>10</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>10</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>10</height>
    </size>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labPerc">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>90</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushBtnTerm</sender>
   <signal>clicked()</signal>
   <receiver>Gengxin</receiver>
   <slot>OnTerm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnOk()</slot>
  <slot>OnCancle()</slot>
  <slot>OnTerm()</slot>
 </slots>
</ui>
